packages/common/src/utils/arrays/utils.ts

```diff
@@ -26,7 +26,7 @@ export const groupBy = <T, K extends keyof T & string>(
   key: K
 ): Record<string, T[]> => {
   const lookup: Record<string, T[]> = {} as Record<string, T[]>;
-  things.forEach(thing => {
+  things?.forEach(thing => {
```

Comment:
offroad fix - was throwing an error if there were no lines

Reply:
lucky me 🤷  Am using the demo server API and one of the invoices ( inbound, I think ) which has no lines was giving an error. just a console error, so no harm done.